provider/diag/diagnostic.go

```diff
@@ -25,6 +25,8 @@ func (d DiagnosticType) String() string {
 		return "Throttle"
 	case DATABASE:
 		return "Database"
+	case USER:
```

Comment:
I know this isn't part of the PR review, but case Internal is missing.

Reply:
fixed